cs/test/core/Util.cs

```diff
@@ -999,12 +999,16 @@ public static IBonded<T> MakeBondedSP<T>(T obj)
                 if (!AnyField<From>(Reflection.IsBonded))
                 {
                     streamTranscode(SerializeSP, TranscodeSPXml<From>, DeserializeXml<To>);
-                
-                    // NewtonSoft JSON doesn't support uint64
+
+#if SUPPORTS_BIGINTEGER
```

Comment:
I misread where the `#if` and `#else` blocks were. This doesn't run the tests twice when there's BigInteger support. If you want to switch, go for it, but no need to.